app/config/locale/translations/en.json

```diff
@@ -36,6 +36,7 @@
     "emails.certificate.footer": "Your previous certificate will be valid for 30 days since the first failure. We highly recommend investigating this case, otherwise your domain will end up without a valid SSL communication.",
     "emails.certificate.thanks": "Thanks",
     "emails.certificate.signature": "{{project}} team",
+    "sms.verification.body": "{{secret}} is your {{project}} verification code.",
```

Comment:
Actually I think its important for the message to be structured as `Your [PROJECT_NAME] verification code is: [CODE]`
This will allow the phone operating systems to detect and suggest codes in apps

Reply:
@christyjacob4, @Meldiron and I researched a bit into how other companies do it and we found that most of them follow this structure where code is in the beginning of the message. This would help in 2 things: 
1. The user can easily read the OTP from the notification without having to open the message
2. If the project name is too large, it would be hard to read the OTP/ the message may exceed allowed characters of one message.

We can finalise the approach and I'll make the changes accordingly.

Reply:
Okay I am fine with either messages as long as its detectable by the OS. Can we do a small test to trigger an SMS and check if the phone recognizes it?